### ISSUE TITLE
Allow aria-required on Elements with the role group

### ISSUE THREAD

**Issue by MarioBatusic** [w3c/aria#2152]
# Description of bug or feature request

There are some use cases that require the "aria-required" attribute to also be allowed on elements with the "group" role. For example, it is a group of input fields, at least one should be filled out - it doesn't matter which one. The other example is a group of checkboxes, of which at least one checkbox must be selected in order to fulfill the required condition.

## Example 1

    <h2 id="ident">Identify Yourself*</h2>

    <div role="group" aria-labelledby="ident" aria-required="true">
      <label for="mailvalue">Mail Address</label> <input type="mail" id="mailvalue"><br>    
      or<br>    
      <label for="user">User Name</label> <input type="text" id="user">
    </div>  

## Example 2

    <h2 id="meals">Choose at least one meal*</h2>

    <div role="group" aria-labelledby="meals" aria-required="true">
      <ul>
        <li><input type="checkbox" id="breakfast"><label for="breakfast">Breakfast</label>    
        <li><input type="checkbox" id="lunch"><label for="lunch">Lunch</label>    
        <li><input type="checkbox" id="dinner"><label for="dinner">Dinner</label>    
      </ul>  
    </div>  

The good news is that JAWS and NVDA already accept this attribute and read it correctly, at least with Chrome and Firefox.

## Will this require a change to [CORE-AAM](https://w3c.github.io/core-aam/)?

If unknown, leave blank. If relevant, link bug.

## Will this require a change to the [ARIA authoring guide](https://w3c.github.io/aria-practices/)?

An additional Example for aria-required could be created.

**Comment by scottaohara** [comment]
link i mentioned during the call - https://github.com/openui/open-ui/issues/487

**Comment by spectranaut** [comment]
Discussed briefly during triage: https://www.w3.org/2024/03/21-aria-minutes.html#t01

**Comment by MarioBatusic** [comment]
OK, I see the rationals, why **aria-required** is not the proper tool for the use cases in this issue:

- Possible conflicts between values set on  **group** role and its children;
- semantic difference between "one is required" and "at least one is required";
- missing meaningful visualisation.
For the use case depicted in this issue we need a new tool.

**Comment by MarioBatusic** [comment]
Will try it on a new way.
